config/meteor.yaml.sample

```diff
@@ -0,0 +1,7 @@
+LOG_LEVEL: hehehehehe
+STATSD_ENABLED: false
+STATSD_HOST: "localhost:8005"
+STATSD_PREFIX: lol
```

Comment:
```suggestion
STATSD_PREFIX: meteor
```

Reply:
sorry, i wasn’t meant to push it, it was just to test the working 😅😅